config/locales/es.yml

```diff
@@ -54,7 +54,6 @@ es:
     formats:
       default: "%-m/%-d/%Y"
       long: "%-d de %B de %Y"
-      medium: "%-d de %B"
```

Comment:
this always gets removed because there is no english equivalent but it should be added back in